Week-4/Assignment-4.2.js

```diff
@@ -0,0 +1,15 @@
+function Person(name) {
+  this.name = name;
+}
+
+function Teacher(name, subject) {
+  Person.call(this, name);
```

Comment:
can you also try with object.setprototype

Reply:
Okay